build/tcpdump/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2020 OmniOS Community Edition.  All rights reserved.
+
+. ../../lib/functions.sh
+
+PROG=tcpdump
+VER=4.9.3
+PKG=ooce/network/test/tcpdump
```

Comment:
```suggestion
PKG=ooce/network/tcpdump
```
Let's drop the `/test` component

Otherwise, looks perfect, thanks!

Reply:
cheers! 
also, I noticed `socat` was missing from `doc/baseline` so I have added
Philip

Reply:
`socat`  is a strange one in that it used to be in the extra repository but it was moved to core in r151031 - see https://github.com/omniosorg/omnios-extra/blob/master/build/socat/build.sh#L26

baseline tracks master which is why it's missing there - otherwise when doing a full extra build on anything newer than r151031, you get an error about a mismatched baseline.

Reply:
thanks, I will keep an eye out for things like that in the future